charts/catalog/README.md

```diff
@@ -2,7 +2,7 @@
 
 Service Catalog is a Kubernetes Incubator project that provides a
 Kubernetes-native workflow for integrating with 
-[Open Service Brokers](https://www.openservicebrokerapi.org/) 
+[Open Service Broker API](http://www.openservicebrokerapi.org/)
```

Comment:
Actually, I think the old text was more correct. Can you change it back to "Brokers"?

Reply:
OK, maybe I understand it incorrectly.